### PR TITLE
docString for numpy.amin

### PR DESCRIPTION
**Pull request by pranavvp16**
also updated the outputs of examples in amax

### CODE REVIEW

File: keras_core/ops/numpy.py [hunk 1]
```diff
@@ -514,6 +514,37 @@ def compute_output_spec(self, x):
...
+        An array with the minimum value. If `axis=None`, the result is a scalar
+        value representing the minimum element in the entire array. If `axis` is
+        given, the result is an array with the minimum values along
+        the specified axis.
+
+    Examples:
+
+    >>> x = keras_core.ops.convert_to_tensor([1, 3, 5, 2, 3, 6])
+    >>> keras_core.ops.amin(x)
+    <tf.Tensor: shape=(), dtype=int32, numpy=1>
```
Comment (AakashKumarNain) [inline keras_core/ops/numpy.py:538]: The output type would depend on the backend being used. For the docstring, It is better to show the output values in a backend agnostic manner

File: keras_core/ops/numpy.py [hunk 2]
```diff
@@ -514,6 +514,37 @@ def compute_output_spec(self, x):
...
+
+    >>> x = keras_core.ops.convert_to_tensor([[1, 6, 8], [7, 5, 3]])
+    >>> keras_core.ops.amin(x, axis=0)
+    <tf.Tensor: shape=(3,), dtype=int32, numpy=array([1, 5, 3], dtype=int32)>
```
Comment (AakashKumarNain) [inline keras_core/ops/numpy.py:542]: Same as above

File: keras_core/ops/numpy.py [hunk 3]
```diff
@@ -514,6 +514,37 @@ def compute_output_spec(self, x):
...
+
+    >>> x = keras_core.ops.convert_to_tensor([[1, 6, 8], [7, 5, 3]])
+    >>> keras_core.ops.amin(x, axis=1, keepdims=True)
+    <tf.Tensor: shape=(2, 1), dtype=int32, numpy= array([[1],[3]], dtype=int32)>
```
Comment (AakashKumarNain) [inline keras_core/ops/numpy.py:546]: Same as above

File: keras_core/ops/numpy.py (hunk 1 above)
Comment (pranavvp16) [reply keras_core/ops/numpy.py]: yeah, understood

**Review by AakashKumarNain — APPROVED** [review]

**Review by fchollet — APPROVED** [review]
LGTM -- thank you!

**Pull request merged**